graphene/types/unmountedtype.py

```diff
@@ -63,7 +63,8 @@ def __eq__(self, other):
         return (
             self is other or (
                 isinstance(other, UnmountedType) and
-                self.get_type() == other.get_type() and
```

Comment:
I wasn't a 100% sure though if equality on the meta class like that just works of if we would need to compare the props directly on the meta class.